apache-rat-core/src/main/java/org/apache/rat/BuilderParams.java

```diff
@@ -33,7 +33,7 @@ public interface BuilderParams {
 
     /**
      * Gets one of the contained methods by name.
-     * 
+     *
```

Comment:
matcherMap() and licenseFamilies() are reported as never used - is there a test case missing or is it called by reflection as well, @Claudenw ?